pkg/ccip/bindings/ownable2step/reader.go

```diff
@@ -0,0 +1,45 @@
+package ownable2step
+
+import (
+	"github.com/xssnick/tonutils-go/address"
+	"github.com/xssnick/tonutils-go/ton"
+
+	"github.com/smartcontractkit/chainlink-ton/pkg/ton/tvm"
+)
+
+var AddressRes = tvm.NewResultDecoder(func(r *ton.ExecutionResult) (*address.Address, error) {
+	ownerSlice, err := r.Slice(0)
+	if err != nil {
+		return nil, err
+	}
+	owner, err := ownerSlice.LoadAddr()
+	if err != nil {
+		return nil, err
+	}
+
+	return owner, nil
+})
+
+func GetOwner(role ...string) tvm.Getter[struct{}, *address.Address] {
+	return tvm.Getter[struct{}, *address.Address]{
+		Name:    prefixGetter("owner", role),
+		Decoder: AddressRes,
+	}
+}
+
+func GetPendingOwner(role ...string) tvm.Getter[struct{}, *address.Address] {
+	return tvm.Getter[struct{}, *address.Address]{
+		Name:    prefixGetter("pendingOwner", role),
+		Decoder: AddressRes,
+	}
+}
```

Comment:
nit: these are not getters but getter generators, so for clarity I'd rename as `MakeGetOwner` and separate default `GetOwner` and `GetPendingOwner` getters.